### ISSUE TITLE
NewWindow sample will effectively hang when non-modal window is closed

### ISSUE THREAD

**Issue by BentTranberg** [elmish/Elmish.WPF#328]
I discovered that the NewWindow sample will effectively appear to be hung when the non-modal extra window is closed. I fixed this by setting `ShutdownMode="OnMainWindowClose"` in the App.xaml. Perhaps you should do the same in the repo.

**Comment by cmeeren** [comment]
To be clear, this happens for me only when I set Window 1 to be either open or hidden, and then close the main window.

Your fix works. Thanks!

**Comment by BentTranberg** [comment]
You are absolutely right - that's exactly what I have observed. I should have stated as much, but my developer mind seems still to be a bit foggy after 9 months of "corona vacation", ending new year. I spent the time well, but not so much on programming.

I'm having a good time rebuilding my WPF application the right way, based on new samples and documentation. Will give feedback when I see something worth bringing up. Again, thanks to you guys for all you're doing.
